.github/workflows/code_checks.yml

```diff
@@ -103,3 +104,11 @@ jobs:
           comment_on_pr: false
           files: log/*.xml
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Run Danger
+        uses: MeilCli/danger-action@v5
+        with:
+          plugins_file: 'Gemfile'
+          install_path: 'vendor/bundle'
+          danger_file: 'Dangerfile'
+          danger_id: 'danger-pr'
```

Comment:
Should move this out into it's own job so something like:

```yaml
danger:
  name: Danger
  env:
    DANGER_GITHUB_API_TOKEN: ${{ secrets.GITHUB_TOKEN }}
  runs-on: ubuntu-20.04
  steps: 
    uses: MeilCli/danger-action@v5
      with:
        plugins_file: 'Gemfile'
        install_path: 'vendor/bundle'
        danger_file: 'Dangerfile'
        danger_id: 'danger-pr'
```
That way it will fail separately from the linting and spec checks

Reply:
done.